src/utilities/HiFiUtilities.ts

```diff
@@ -186,3 +186,40 @@ export function preciseInterval(callback: Function, intervalMS: number): any {
     let timeout:any = setTimeout(wrapper);
     return { clear: () => clear(timeout) };
 }
+
+export function checkBrowserCompatibility(): Boolean {
+    let requiredFeatures: Array<string> = [
+        // Navigator mediaDevices
+        "navigator",
+        "navigator.permissions",
+        "navigator.permissions.query",
+        "navigator.mediaDevices.getUserMedia",
+        "navigator.mediaDevices.getSupportedConstraints",
+        // WebRTC
+        "window.MediaStream",
+        "window.MediaStreamTrack",
+        "window.RTCDataChannel",
+        "window.RTCDataChannelEvent",
+        "window.RTCDtlsTransport",
+        "window.RTCIceCandidate",
+        "window.RTCIceTransport",
+        "window.RTCPeerConnection",
+        "window.RTCPeerConnectionIceEvent",
+        "window.RTCRtpReceiver",
+        "window.RTCRtpSender",
+        "window.RTCRtpTransceiver",
+        "window.RTCSctpTransport",
+        "window.RTCSessionDescription"
+    ]
+    for (let i = 0; i < requiredFeatures.length; i++) {
+        if (typeof(eval(requiredFeatures[i])) === "undefined") {
+            console.log("HiFi Audio API: The browser does not support: " + requiredFeatures[i]);
+            if (requiredFeatures[i] === "navigator.mediaDevices.getUserMedia") {
+                console.log("HiFi Audio API: It might be preventing access to this feature on insecure contexts.")
```

Comment:
Please replace with `HiFiLogger.error()`, i.e.:
```suggestion
                HiFiLogger.error("HiFi Audio API: It might be preventing access to this feature on insecure contexts.")
```

Reply:
Also, perhaps consider this language:
```suggestion
                HiFiLogger.error("HiFi Audio API: Your browser may be preventing access to this feature if you are running in an insecure context, i.e. an `http` server.")
```

Reply:
Yeah that reads better. Changed